abb_bringup/launch/abb_moveit.launch.py

```diff
@@ -32,94 +30,47 @@ def launch_setup(context, *args, **kwargs):
     moveit_config_package = LaunchConfiguration("moveit_config_package")
     moveit_config_file = LaunchConfiguration("moveit_config_file")
 
-    # Planning context
-    robot_description_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(support_package), "urdf", robot_xacro_file]
-            ),
-        ]
-    )
-    robot_description = {"robot_description": robot_description_content}
-
-    robot_description_semantic_content = Command(
-        [
-            PathJoinSubstitution([FindExecutable(name="xacro")]),
-            " ",
-            PathJoinSubstitution(
-                [FindPackageShare(moveit_config_package), "config", moveit_config_file]
-            ),
-        ]
+    # MoveIt configuration
+    moveit_config = (
+        MoveItConfigsBuilder("abb_irb1200_5_90")
+        .robot_description(file_path=os.path.join(
+            get_package_share_directory(f'{support_package.perform(context)}'),
+            'urdf', f'{robot_xacro_file.perform(context)}'))
+        .robot_description_semantic(file_path=os.path.join(
+            get_package_share_directory(f'{moveit_config_package.perform(context)}'),
+            'config', f'{moveit_config_file.perform(context)}'))
+        .planning_pipelines(
+            pipelines=["ompl"],
+            default_planning_pipeline="ompl",)
+        .robot_description_kinematics(file_path=os.path.join(
```

Comment:
Nit: Technically apart from the UDRF and SRDF, the builder will automatically load the `kinematics.yaml`, `moveit_controllers.yaml`, `joint_limits.yaml` files since we have specific the package name. So we can remove these entries along with the `.planning_scene_mointor` which will be initialized to same by default. But I'm okay with leaving as is if you prefer.